index.html

```diff
@@ -34,23 +33,36 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=InaiMathi"
     />
-    <title>Document</title>
+    <title>Sangwa Portfolion Home page</title>
   </head>
   <body>
     <header>
       <nav class="nav-bar">
-        <div>Sangwa</div>
+        <div class="name">Sangwa</div>
+        <div class = "mobile-menu">
+          <img class = "close-icon" src="img/Icon.svg" alt="" onclick="mobileMenu()"> 
+          <div class="menu-list">
+            <ul>
+              <li>Portfolio</li>
+              <li>About</li>
+              <li>Contact</li>
+            </ul>
+          </div>
+        </div>
         <div class="menu">
-          <img
-            src="https://img.icons8.com/color/48/000000/menu--v5.png"
-            alt=""
-          />
+          <div>
+            <a href="javascript:void(0);" id = "menu-button">
+            <img src="https://img.icons8.com/color/48/000000/menu--v5.png"
+            alt="Menu icon for mobile view" onclick="mobileMenu()"
+          /> 
```

Comment:
> You have done an amazing job in this project setting up both, JS and CSS. But there are some issues related to the project requirements I will list them below. 

- [ ] Please make sure that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is be displayed. Currently there are no links inside `li` tags. 
- [ ] Please make sure that when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

Reply:
Addressed ✔